pkg/util/runtime/features.go

```diff
@@ -100,6 +100,22 @@ var (
 	// * change the value in site/content/en/docs/Guides/feature-stages.md.
 	// * Ensure that the features in each file are organized categorically and alphabetically.
 	//
+	// Feature Promotion: alpha->beta for SDK Functions
+	// * Move Files from alpha->beta:
+	// 		- From proto/sdk/alpha/alpha.proto to proto/sdk/beta/beta.proto
+	//		- For each language-specific SDK (e.g., Go, C#, Rust):
+	//			- Move implementation files (e.g., alpha.go to beta.go)
+	//			- Move test files (e.g., alpha_test.go to beta_test.go)
+	//			- Note: Delete references to 'alpha' in the moved alpha files.
+	// * Change all instances of alpha->beta:
+	//		- Proto Files: proto/sdk/sdk.proto
+	//		- SDK Implementations: Update in language-specific SDKs (e.g., sdks/go/sdk.go, sdks/csharp/sdk/AgonesSDK.cs).
```

Comment:
This seems redundant to what you have above?

Reply:
The first steps are for moving alpha feature from alpha files to beta files and this step is for change/replace alpha to beta.

Reply:
It will be easy to spot the error if we run the `make gen-all-sdk-grpc` and make run-sdk-conformance-test-go` commands right?